export_csv_verwaltungsgebiete.sh

```diff
@@ -29,46 +28,51 @@ EOF
 function build_static_query {
   local query=$(cat << EOF
     SELECT
-      ags,
-      null as parent_ags,
+      ars,
+      null as parent_ars,
       'sta' as type,
       gen,
-      bez,
-      geom
+      bez
     FROM
       vg250_sta
     WHERE
-      ags = '${1}'
+      ars = '${1}'
     AND gf = '4'
 EOF
   )
   echo ${query}
 }
 
 function export_to_csv {
-  local ags=${1}
+  local ars=${1}
   local query=${2}
-  echo "Starting export for ${ags}"
+  echo "Starting export for ${ars}"
   psql\
     -h localhost\
     -d postgis\
     -U docker\
-    -c "Copy ($query) To '/app/data/${ags}.csv' With CSV DELIMITER ',' HEADER;"
-  echo "Exported to data/${ags}.csv"
+    -c "Copy ($query) To '/app/data/${ars}.csv' With CSV DELIMITER ',' HEADER;"
+  echo "Exported to data/${ars}.csv"
+}
+
+function ensure_data_dir {
+  mkdir -p /app/data
```

Comment:
Normally this folder does exist from the import (if started as mentioned in the readme). But it's more safe like this.